packages/x-data-grid-premium/src/hooks/features/aggregation/gridAggregationInterfaces.ts

```diff
@@ -74,7 +74,7 @@ export interface GridAggregationFunction<V = any, AV = V> {
   getCellValue?: (params: GridAggregationGetCellValueParams) => V;
 }
 
-interface GridAggregationParams<V = any> {
+export interface GridAggregationParams<V = any> {
```

Comment:
Why?

Reply:
I think you had an outdated `master` branch in your MUI X fork.
I'll merge master into it 👍🏻